src/lib/util/Games/WheelOfFortune.ts

```diff
@@ -96,12 +97,17 @@ export class WheelOfFortune {
 		const darkTheme = settings.get(UserSettings.DarkTheme);
 		const spin = Math.floor(Math.random() * this.MULTIPLIERS.length);
 		this.calculate(spin);
+		const won = this.winnings === 0;
+		const winnings = (this.winnings * this.boost) - this.amount;
 
-		const amount = this.winnings === 0 ? money - this.amount : money - this.amount + (this.winnings * this.boost);
+		const amount = won
+			? money - this.amount
+			: money + winnings;
 		if (amount < 0) throw this.message.language.tget('GAMES_CANNOT_HAVE_NEGATIVE_MONEY');
-		await settings.update(UserSettings.Money, amount, {
-			extraContext: { author: this.message.author.id }
-		});
+
+		await (won
+			? this.player.decreaseBalance(this.amount, EconomyTransactionReason.Gamble)
+			: this.player.increaseBalance(winnings, EconomyTransactionReason.Gamble));
```

Comment:
You know, the code is 1:1 to before, but `won ? decrease : increase` is... misleading.
I suggest renaming `won` to `lost`, or change `won` to be `this.winnings > 0;`